target/linux/ipq40xx/files-5.4/arch/arm/boot/dts/qcom-ipq4018-fritzbox-4040.dts

```diff
@@ -110,33 +100,33 @@
 
 		wlan {
 			label = "green:wlan";
-			gpios = <&qca8075 1 GPIO_ACTIVE_HIGH>;
+			gpios = <&ethphy0 1 GPIO_ACTIVE_HIGH>;
```

Comment:
As you mentioned, after correcting `gc->ngpio` , `1 (0x8074 + 1)` here should be `0 (0x8075 + 0 * 2)`

Reply:
Thanks, this should work as on AP120C the conversion works.